File: src/main/java/dev/tang/tool/util/TimeUtils.java
package dev.tang.tool.util;

import java.text.SimpleDateFormat;
import java.util.Date;

public class TimeUtils {

    /**
     * Get current time
     */
    public static String getCurrentTime() {
        long now = System.currentTimeMillis();
        SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
        return format.format(new Date(now));
    }
}

File: src/main/java/dev/tang/tool/util/TimeUtils.kt
package dev.tang.tool.util

import java.text.SimpleDateFormat
import java.util.*

object TimeUtils {

    /**
     * Get current time
     */
    val currentTime: String
        get() {
            val now = System.currentTimeMillis()
            val format = SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
            return format.format(Date(now))
        }
}
